cpp/modmesh/toggle/profile.cpp

```diff
@@ -36,31 +36,46 @@ std::string TimeRegistry::detailed_report() const
     std::ostringstream ostm;
     /// Header
     ostm
-        << std::setw(40) << total_call_count()
+        << std::setw(50) << total_call_count()
```

Comment:
Add 10 blanks before the header to set it at the center.

Reply:
Maybe you can consider making all these numbers as constant values.

Reply:
`constexpr` sounds like a good idea at the moment.  The code looks like:

```cpp
constexpr int width = 50;
ostm
     << std::setw(width) << total_call_count();
```

Reply:
The constant indeed makes the code clearer. I'll add it to the code.